django/contrib/admin/checks.py

```diff
@@ -924,6 +938,14 @@ def _check_list_display_item(self, obj, item, label):
                     id="admin.E109",
                 )
             ]
+        elif isinstance(field, related.ReverseManyToOneDescriptor):
+            return [
+                checks.Error(
+                    "The value of '%s' must not be a reverse related field." % label,
+                    obj=obj.__class__,
+                    id="admin.E413",
```

Comment:
I'd reuse `E109` instead of raising a different error, and check attributes rather than `isinstance()`, e.g.
```diff
diff --git a/django/contrib/admin/checks.py b/django/contrib/admin/checks.py
index d64228557e..d58d755e0a 100644
--- a/django/contrib/admin/checks.py
+++ b/django/contrib/admin/checks.py
@@ -930,7 +930,10 @@ class ModelAdminChecks(BaseModelAdminChecks):
                         id="admin.E108",
                     )
                 ]
-        if isinstance(field, models.ManyToManyField):
+        if (
+            isinstance(field, models.ManyToManyField)
+            or (getattr(field, "rel", False) and field.rel.field.many_to_one)
+        ):
             return [
                 checks.Error(
                     "The value of '%s' must not be a ManyToManyField." % label,

```

Reply:
Done